master/internal/experiment.go

```diff
@@ -594,7 +588,7 @@ func (e *experiment) setPriority(ctx *actor.Context, priority *int) error {
 		return errors.Wrapf(err, "setting experiment %d priority", e.ID)
 	}
 	ctx.Tell(sproto.GetRM(ctx.Self().System()), job.SetGroupPriority{
-		Priority: priority,
+		Priority: *priority,
```

Comment:
should we also change this function to also not take a pointer?

Reply:
This one was hard to do because he priority being modified is in the config resources, which I'm reluctant to change yet. The code actually gets more complex if we don't pass in the pointer.